util/election2/testonly/tests.go

```diff
@@ -64,7 +64,7 @@ func checkDone(ctx context.Context, t *testing.T, wait time.Duration) {
 
 // runElectionAwait tests the Await call with different pre-conditions.
 func runElectionAwait(t *testing.T, f election2.Factory) {
-	awaitErr := errors.New("Await error")
+	awaitErr := errors.New("await error")
```

Comment:
Not sure about this one, "Await" here means the name of the exported method. But should be fine since this is a test.

Reply:
It's only a placeholder error for the test. I think it will do as is.